client/src/modules/home/home-services/plan.service.js

```diff
@@ -3,5 +3,25 @@ export default class PlanService {
     this.$inject = ['$http'];
 
   }
+  getPlan(endpoint, params, callback) {
+    $http({
+      method: 'GET',
+      url: endpoint,
+      params: params
+    }).then(function successCallback(response) {
+      callback(response.data);
+    }, function errorCallback(response) {

```

Comment:
this is outdated Angular promise callback syntax, it took me forever to find a reference where they said it (it's not in the official docs), but as of 1.4 or 1.5 they now support (and recommend) the es6, more modern approach with:

.then(resp => do something).catch(err => do something)